src/js/components/DataChart/Detail.js

```diff
@@ -224,7 +224,7 @@ const Detail = ({
           <Box pad="small" background={{ color: 'background-back' }}>
             <Grid
               columns={['auto', 'auto', 'auto']}
-              gap="xsmall"
+              gap={theme.dataChart.detail.gap}
```

Comment:
It looks like in the theme we define `theme.dataChart.detail.grid.gap` but here we just point to `theme.dataChart.detail.gap`